components/config-mgmt-service/integration_test/suggestions_test.go

```diff
@@ -44,6 +44,355 @@ func TestSuggestionsWithAnInvalidTypeReturnsError(t *testing.T) {
+		"nginx.geoip.lib_checksum",
+		"git.url",
+		"apt.confd.install_suggests",
+		"apt.unattended_upgrades.minimal_steps",
+		"yum.main.deltarpm",
+		"nginx.worker_connections",
+		"nginx.upstart.foreground",
+		"apt.caching_server",
+		"apt.unattended_upgrades.mail_only_on_error",
+		"rsyslog.tls_certificate_file",
+		"opscode-ci.monitoring.jmx_hostname",
+		"nginx.upload_progress.zone_size",
+		"nginx.source.sbin_path",
+		"yum.epel-source.gpgkey",
+		"nginx.echo.url",
+		"nginx.gzip_min_length",
+		"authorization.sudo.env_keep_subtract",
+		"nginx.rate_limit",
+		"yum.main.sslverify",
+		"apt.unattended_upgrades.package_blacklist",
+		"yum.main.throttle",
+		"opscode-ci.master.days_to_keep_artifacts",
+		"nginx.accept_mutex_delay",
+		"nginx.passenger.ruby",
+		"omnibus.build_user_group",
+		"omnibus.build_user_home",
+		"jenkins.master.endpoint",
+		"git.version",
+		"git.checksum",
+		"authorization.sudo.sudoers_defaults",
+		"tags",
+		"omnibus.build_user",
+		"omnibus.ruby_version",
+		"omnibus.build_user_password",
+		"authorization.sudo.include_sudoers_d",
+		"authorization.sudo.agent_forwarding",
+		"opscode-ci.omnitruck.metadata_bucket",
+		"opscode-ci.omnitruck.packages_bucket",
+		"opscode-ci.master.pipeline_types",
+		"opscode-ci.master.host_name",
+		"opscode-ci.master.days_to_keep_artifacts",
+		"opscode-ci.master.artifacts_to_keep",
+		"opscode-ci.artifactory.endpoint",
+		"yum.main.clean_requirements_on_remove",
+		"yum.main.plugins",
+		"nginx.lua.version",
+		"yum.main.obsoletes",
+		"aws.databag_name",
+		"windows.rubyzipversion",
+		"rsyslog.default_facility_logs.uucp,news.crit",
+		"nginx.headers_more.source_checksum",
+		"nginx.upstart.foreground",
+		"bluepill.logfile",
+		"yum.main.sslclientcert",
+		"apt.cacher_port",
+		"apt.confd.install_recommends",
+		"nginx.log_dir_perm",
+		"xml.nokogiri.use_system_libraries",
+		"yum.epel.gpgkey",
+		"yum.main.color_list_installed_extra",
+		"nginx.naxsi.checksum",
+		"nginx.passenger.packages.rhel",
+		"nginx.echo.url",
+		"yum.main.password",
+		"yum.main.color_list_available_reinstall",
+		"aws.databag_entry",
+		"omnibus.install_dir",
+		"nginx.passenger.gem_binary",
+		"yum.epel-testing-debuginfo.description",
+		"authorization.sudo.include_sudoers_d",
+		"nginx.upload_progress.checksum",
+		"nginx.source.use_existing_user",
+		"nginx.source.version",
+		"nginx.binary",
+		"apt.cacher_dir",
+		"homebrew.auto-update",
+		"rsyslog.service_name",
+		"nginx.passenger.packages.fedora",
+		"opscode-ci.email.list_id",
+		"aws.right_aws_version",
+		"nginx.geoip.country_dat_checksum",
+		"nginx.pagespeed.packages.rhel",
+		"rsyslog.tls_ca_file",
+		"packagecloud.gpg_key_path",
+		"yum.epel-debuginfo.managed",
+		"yum.main.logfile",
+		"runit.executable",
+		"windows.allow_reboot_on_failure",
+		"nginx.set_misc.version",
+		"nginx.passenger.packages.debian",
+		"nginx.auth_request.url",
+		"yum.epel-testing-debuginfo.make_cache",
+		"zum.epel-debuginfo.gpgcheck",
+	}
```

Comment:
You could save lines of code and make this easier to read/understand (maybe?) if you looped the number of times and built strings like `z1`, `z2`, etc. I'm thinking it'd maybe be easier to understand a failure if you got back just the wrong string rather than an arbitrary one from your collection.